src/Bathymetry.jl

```diff
@@ -115,12 +90,9 @@ Keyword Arguments
                   the smallest basins are removed first. `major_basins = 1` retains only the largest basin.
                   If `Inf` then no basins are removed. Default: 1.
 """
-function regrid_bathymetry(target_grid;
+function regrid_bathymetry(target_grid, m::Metadatum;
```

Comment:
```suggestion
function regrid_bathymetry(target_grid, metadatum::Metadatum;
```
a little more verbose